tests/nodes/NodeConnectionManager.lifecycle.test.ts

```diff
@@ -323,21 +354,17 @@ describe(`${NodeConnectionManager.name} lifecycle test`, () => {
         },
       );
 
-      // Connection is not created yet, no locking applied
+      // Connection is not created yet
       expect(connections.get(remoteNodeIdString1)).not.toBeDefined();
 
       // Iterating over generator
       for await (const _ of gen) {
-        // Should be locked for duration of stream
-        expect(connectionLocks.isLocked(remoteNodeIdString1)).toBe(true);
+        // Do nothing
       }
```

Comment:
Same here, why run this if it does nothing?

Reply:
Same reason as above but for `withConnG`.